extensions/amp-skimlinks/0.1/skim-options.js

```diff
@@ -147,6 +150,30 @@ function getInternalDomains_(docInfo) {
   return internalDomains;
 }
 
+/**
+ * Use CNAME domain defined in "custom-redirect-domain" option if specified
+ * or "https://go.skimresources.com" by default.
+ * @param {!Element} element
+ * @return {string}
+ */
+function getWaypointBaseUrl(element) {
+  let customSubDomain = element.getAttribute('custom-redirect-domain');
+  if (customSubDomain) {
+    // Remove potential HTTP protocol.
+    customSubDomain = customSubDomain.replace(/^\/\/|^https?:\/\//, '');
+    // Remove potential trailing slash.
+    customSubDomain = endsWith(customSubDomain, '/') ?
```

Comment:
just add the removal of the potential trailing '/' case to the regex and then update the comment. 

    // Remove potention HTTP protocol and potential trailing slash.
    customSubDomain = customSubDomain.replace(/^\/\/|^https?:\/\/|\/$/g, '');